README.md

```diff
@@ -142,6 +169,81 @@ To change this behavior use `skipOnEmpty(true)`:
 Number::rule()->integer()->max(100)->skipOnEmpty(true)
 ```
 
+#### Nested and related data
+
```

Comment:
```suggestion
#### Nested and related data

In many cases there is a need to validate data consisting of many similar structures. There is a `Nested` rule for this purpose:
```

Reply:
Good addition. But I think I'll correct this a bit. `Nested` is more about relations, while `Each` - about similar structures.

Reply:
:heavy_check_mark: